LowestCommon.py

```diff
@@ -0,0 +1,17 @@
+"""
+The approach here is to check with the root node value and p and q value everytime since it's a bst left 
+side elements would be less and right would be greater than root, so at every point we check if p is less
+than root and q is greater than root or viceversa, id so we return the particular root val as the answer,
+else if both p and q are greater than we change our root to right node of root or else if both are smaller
+than to the left node.
```

Comment:
great explanation patil!!
this is your plus point...the way you explain your approaches is amazing!!